csrc/runtime/fusion_kernel_runtime.cpp

```diff
@@ -517,8 +517,8 @@ std::optional<std::unique_ptr<HeuristicParamsList>> FusionKernelRuntime::
               group_to_run, fusion_to_run_info);
     } else {
       // Try to get scheduler entry
-      auto maybe_heuristic_params =
-          group_to_run->getMaybeHeuristicParams(fusion_to_run_info);
+      auto maybe_heuristic_params = group_to_run->getMaybeHeuristicParams(
+          fusion_to_run_info, /*skip_compile_time_checks=*/true);
```

Comment:
Can you add a comment about why compile time checks can be skipped and should be skipped?